packages/iris-grid/src/AdvancedFilterCreator.tsx

```diff
@@ -495,17 +585,19 @@ class AdvancedFilterCreator extends PureComponent {
           {isValuesTableAvailable && !valuesTableError && (
             <>
               {!isBoolean && <hr />}
-              <div className="form-group">
-                <AdvancedFilterCreatorSelectValue
-                  table={valuesTable}
-                  onChange={this.handleSelectValueChange}
-                  invertSelection={invertSelection}
-                  selectedValues={selectedValues}
-                  formatter={formatter}
-                  showSearch={!isDateType}
-                  timeZone={formatter.timeZone}
-                />
-              </div>
+              {valuesTable && (
```

Comment:
We should _probably_ put a spinner here while the valuesTable is loading... something like:
```
{ !valuesTable && <LoadingSpinner ... />}
```

Not necessary as part of this change though.